src/components/App.tsx

```diff
@@ -258,8 +258,8 @@ export class App extends React.Component<AppProps, AppState> {
     if (!this.state.accountId) {
       const randomSuffix = Math.floor(Math.random() * 9999999999);
       const accountId = "studio" + randomSuffix;
-      const keyPair = await KeyPair.fromRandom("ed25519");
-      await createAccount(accountId, keyPair.getPublicKey());
+      const keyPair = await KeyPair.fromRandom();
+      await createAccount(accountId, keyPair.getPublicKey().toString());
```

Comment:
why is this change needed?

Reply:
It wouldn't build for me otherwise.  The API in KeyPair was different.